presto-jdbc/src/main/java/com/facebook/presto/jdbc/QueryExecutor.java

```diff
@@ -83,19 +129,45 @@ protected void finalize()
         close();
     }
 
-    static QueryExecutor create(String userAgent)
+    static Properties filterClientProperties(Properties connectionProperties)
```

Comment:
1. We'll need it as soon as we add the truststore stuff
2. The new code also turns the Properties into an ImmutableMap per @arhimondr's suggestion